ClemBot.Bot/bot/api/tag_route.py

```diff
@@ -1,67 +1,115 @@
 import typing as t
-
 from bot.api.api_client import ApiClient
 from bot.api.base_route import BaseRoute
+from dataclasses import dataclass
+from dataclasses_json import LetterCase, DataClassJsonMixin, dataclass_json
+
+
+@dataclass_json(letter_case=LetterCase.CAMEL)
+@dataclass
+class Tag(DataClassJsonMixin):
+    name: str
+    content: str
+    creation_date: str
+    guild_id: int
+    user_id: int
+    use_count: int
```

Comment:
Because we also use this in the cogs we should pull the models out into a top level folder called "models" that way they can be used in both the cogs and the routes

Reply:
Right now I have it to where we would put all models in a single py file called `modules.py` that's under the `bot` folder:
```py
from dataclasses import dataclass
from dataclasses_json import LetterCase, DataClassJsonMixin, dataclass_json


@dataclass_json(letter_case=LetterCase.CAMEL)
@dataclass
class Tag(DataClassJsonMixin):
    name: str
    content: str
    creation_date: str
    guild_id: int
    user_id: int
    use_count: int = 0


@dataclass_json(letter_case=LetterCase.CAMEL)
@dataclass
class Infraction(DataClassJsonMixin):
    id: int
    guild_id: int
    author_id: int
    subject_id: int
    type: str
    reason: str
    duration: int
    time: str
    active: int
```
So importing would just be `from bot.models import Tag` or `from bot.models import Infraction`. Is this good?

Reply:
hmm, ya this works